src/services/notificationservice.cpp

```diff
@@ -17,6 +17,16 @@
 
 #include "notificationservice.h"
 
+const static struct VibrateSymbols {
+    NotificationService::Vibrate symbol;
```

Comment:
I'd recommend:
```
static const char *VibrateStr[] = { "ringtone", "strong", "normal", "none" };
```
In the header instead. Notice how:
- this does not need a new struct
- this does not need an array size (if someone adds a value to the enum I can guarantee you they won't think of changing the size of this array)
- this would keep the enum and the associated strings in the same place so that it's more obvious this needs to be changed when the enum changes and it's more obvious how the index in the array maps to the enum value

Reply:
I decided eliminate the enum entirely and simply pass a QString, simplifying the interface considerably.  There is no longer any error checking but there already was no good way to return errors anyway.